Modules/ThirdParty/GDCM/src/gdcm/CMakeLists.txt

```diff
@@ -353,7 +353,7 @@ else()
 endif()
 
 if(GDCM_USE_SYSTEM_OPENJPEG)
-  find_package(OpenJPEG 2.0.0 REQUIRED)
+  #find_package(OpenJPEG  REQUIRED)
   set(GDCM_OPENJPEG_LIBRARIES ${OPENJPEG_LIBRARIES})
```

Comment:
If `USE_SYSTEM_JPEG` is enabled in ITK, will this variable have the right contents?

Reply:
USE_SYSTEM_JPEG refer to the libjpeg/libjpeg-turbo not openjpeg. There currently is not support for a system openjpeg.